gui/src/containers/JobSpec.js

```diff
@@ -193,9 +208,6 @@ const mapStateToProps = (state, ownProps) => {
   }
 }
 
-export const ConnectedJobSpec = connect(
-  mapStateToProps,
-  matchRouteAndMapDispatchToProps({fetchJobSpec})
-)(JobSpec)
+export const ConnectedJobSpec = connect(mapStateToProps, matchRouteAndMapDispatchToProps({ fetchJobSpec }))(JobSpec)
```

Comment:
I prefer the previous multi line style

Reply:
I don't make these changes. It's vscode :(
I'll revert this